fenix/app/src/main/java/org/mozilla/fenix/settings/SecretDebugSettingsFragment.kt

```diff
@@ -39,41 +42,46 @@ class SecretDebugSettingsFragment : Fragment() {
         return ComposeView(requireContext()).apply {
             setContent {
                 FirefoxTheme {
-                    DebugInfo()
+                    DebugInfo(components.core.store)
                 }
             }
         }
     }
 }
 
 @Composable
-private fun DebugInfo() {
-    val store = components.core.store
+private fun DebugInfo(browserStore: BrowserStore) {
```

Comment:
I don't think we should ever pass `BrowserStore` as a parameter to our Composables. Could we observe the `regionState` above and pass this state instead?

Reply:
We can certainly, extracted a top level composable that just observes the state and passes the `regionState` to `DebugInfo()`